src/components/ProfilePage/ProfilePageHeader/ProfilePageHeader.jsx

```diff
@@ -53,13 +53,15 @@ export default class extends React.Component {
                         {/* props.subtext */}
                     </span>
                     <div className='profileTextInformationNetworkProgressContainer'>
+                        {/*
```

Comment:
Kan egentlig ha med stjernerating for å vise hvordan det skal se ut. Bare sett det til 4 stjerner eller noe sånt. Blir litt tomt uten det.